{"text": "2024-06-16 06:30:13 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.060177", "seconds": 1.060177}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic/epic/epic/src/claim.py"}, "function": "stash", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:30:13.478952+08:00", "timestamp": 1718490613.478952}}}
{"text": "2024-06-16 06:30:21 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:09.457004", "seconds": 9.457004}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic/epic/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:30:21.875779+08:00", "timestamp": 1718490621.875779}}}
{"text": "2024-06-16 06:30:24 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:00:12.249159", "seconds": 12.249159}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:30:24.667934+08:00", "timestamp": 1718490624.667934}}}
{"text": "2024-06-16 06:30:26 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0x9d in position 0: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic/epic/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0x9d in position 0: invalid start byte\n", "record": {"elapsed": {"repr": "0:00:14.302476", "seconds": 14.302476}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0x9d in position 0: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0x9d in position 0: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:30:26.721251+08:00", "timestamp": 1718490626.721251}}}
{"text": "2024-06-16 06:30:28 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:00:16.520973", "seconds": 16.520973}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:30:28.939748+08:00", "timestamp": 1718490628.939748}}}
{"text": "2024-06-16 06:30:58 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:00:46.071271", "seconds": 46.071271}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:30:58.490046+08:00", "timestamp": 1718490658.490046}}}
{"text": "2024-06-16 06:30:58 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:00:46.071814", "seconds": 46.071814}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:30:58.490589+08:00", "timestamp": 1718490658.490589}}}
{"text": "2024-06-16 06:31:29 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:01:17.152488", "seconds": 77.152488}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:31:29.571263+08:00", "timestamp": 1718490689.571263}}}
{"text": "2024-06-16 06:31:32 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:01:19.931050", "seconds": 79.93105}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:31:32.349825+08:00", "timestamp": 1718490692.349825}}}
{"text": "2024-06-16 06:31:34 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xa9 in position 0: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic/epic/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xa9 in position 0: invalid start byte\n", "record": {"elapsed": {"repr": "0:01:21.962526", "seconds": 81.962526}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xa9 in position 0: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xa9 in position 0: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:31:34.381301+08:00", "timestamp": 1718490694.381301}}}
{"text": "2024-06-16 06:31:36 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:01:24.293694", "seconds": 84.293694}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:31:36.712469+08:00", "timestamp": 1718490696.712469}}}
{"text": "2024-06-16 06:32:06 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:01:53.843271", "seconds": 113.843271}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:32:06.262046+08:00", "timestamp": 1718490726.262046}}}
{"text": "2024-06-16 06:32:06 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:01:53.843914", "seconds": 113.843914}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:32:06.262689+08:00", "timestamp": 1718490726.262689}}}
{"text": "2024-06-16 06:32:37 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:02:24.933445", "seconds": 144.933445}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:32:37.352220+08:00", "timestamp": 1718490757.35222}}}
{"text": "2024-06-16 06:32:40 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:02:27.693859", "seconds": 147.693859}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:32:40.112634+08:00", "timestamp": 1718490760.112634}}}
{"text": "2024-06-16 06:32:42 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0x81 in position 1: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic/epic/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0x81 in position 1: invalid start byte\n", "record": {"elapsed": {"repr": "0:02:29.643795", "seconds": 149.643795}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0x81 in position 1: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0x81 in position 1: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:32:42.062570+08:00", "timestamp": 1718490762.06257}}}
{"text": "2024-06-16 06:32:44 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:02:31.978913", "seconds": 151.978913}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:32:44.397688+08:00", "timestamp": 1718490764.397688}}}
{"text": "2024-06-16 06:33:13 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:03:01.528644", "seconds": 181.528644}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:33:13.947419+08:00", "timestamp": 1718490793.947419}}}
{"text": "2024-06-16 06:33:13 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:03:01.529378", "seconds": 181.529378}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:33:13.948153+08:00", "timestamp": 1718490793.948153}}}
{"text": "2024-06-16 06:33:45 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:03:32.620370", "seconds": 212.62037}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic/epic/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:33:45.039145+08:00", "timestamp": 1718490825.039145}}}
{"text": "2024-06-16 06:33:45 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3684), thread 'MainThread' (139958461406080): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic/epic/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic/epic/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic/epic/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic/epic/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic/epic/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:03:32.773449", "seconds": 212.773449}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic/epic/epic/src/claim.py"}, "function": "run", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3684), thread 'MainThread' (139958461406080):", "module": "claim", "name": "__main__", "process": {"id": 3684, "name": "MainProcess"}, "thread": {"id": 139958461406080, "name": "MainThread"}, "time": {"repr": "2024-06-16 06:33:45.192224+08:00", "timestamp": 1718490825.192224}}}
